src/app/shared/header.js

```diff
@@ -18,7 +18,12 @@ class Header extends React.Component {
 		<div id ='header-section' className=' content-container'>
       <header>
         <img src='http://www.colorado.edu/cwa/profiles/express/themes/culayers/logo.png' />
-        <h1>Volunteer Management System</h1>
+          <div className="header__name-and-slogan" id="name-and-slogan">
```

Comment:
Bit of a Nitpick: Let's maintain our use of "-" rather than "_" for all CSS class names.